desktop/components/main_layout/footer/lower.jade

```diff
@@ -1,14 +1,18 @@
 .mlf-lower: .main-layout-container
   nav#main-layout-footer-internal
-    a#main-layout-footer-logo.icon-logo( href='/' )
+    a#main-layout-footer-logo( href='/' )
+      include ../public/icons/logo.svg
```

Comment:
If the error is higher up the stack than [bucketAssets](https://github.com/artsy/force/blob/master/lib/setup.js#L182) the icon-font files won't load properly and you'll see square boxes. Another point for to inline SVG!